doc/btp_aics.txt

```diff
@@ -19,75 +19,238 @@ Commands and responses:
 
 		In case of an error, the error response will be returned.
 
-	Opcode 0x02 - Set gain
-		Controller Index:	<controller id>
-		Command parameters:	Gain (1 octet)
-		Response parameters:	<none>
+	Opcode 0x02 - Set Gain
+	    Controller Index:   <controller id>
```

Comment:
To keep the formatting of documentation files consistent, please review them again and use tabs instead of spaces.
```suggestion
		Controller Index:	<controller id>
```